Hippo.Web/ClientApp/src/components/Order/CreateOrder.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { useEffect, useState } from "react";
+import { OrderModel } from "../../types";
+import { useParams } from "react-router-dom";
+import { usePermissions } from "../../Shared/usePermissions";
+import { usePromiseNotification } from "../../util/Notifications";
+import OrderForm from "./OrderForm";
+import { authenticatedFetch, parseBadRequest } from "../../util/api";
+
+const defaultOrder: OrderModel = {
```

Comment:
would be nice to have in different file, but can be moved with later refactoring

Reply:
Not sure about this one.

Reply:
i usually just put this in like a `Order.ts` file or wherever the model is